src/app/codegen-data-model-provider/CodegenDataModelProvider_Write.cpp

```diff
@@ -356,6 +357,8 @@ DataModel::ActionReturnStatus CodegenDataModelProvider::WriteAttribute(const Dat
             //       however for now this is called directly because ember code does this call
             //       inside emberAfWriteAttribute.
             MatterReportingAttributeChangeCallback(request.path);
+
+            emberAfIncreaseClusterDataVersion(request.path);
```

Comment:
This increases the data version twice.  Why do we want that?

Reply:
We only want one call. Removed the reporting callback as the `increase version + dirty mark` should be the equivalent.